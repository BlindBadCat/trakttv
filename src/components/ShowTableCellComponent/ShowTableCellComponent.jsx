import React from 'react';

const ShowTableCellComponent = ({content}) => <td style={{fontSize:25}}>{content}</td>

export default ShowTableCellComponent;
