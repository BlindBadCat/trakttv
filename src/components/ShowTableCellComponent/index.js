import ShowTableCellComponent from './ShowTableCellComponent'

export default ShowTableCellComponent;