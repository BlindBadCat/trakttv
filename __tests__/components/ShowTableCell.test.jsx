import React from 'react';
import { shallow } from 'enzyme';
import ShowTableCellComponent from '../../src/components/ShowTableCellComponent';

describe('<ShowTableCellComponent />', () => {
  it('Should render content', () => {
    const cell = shallow(<ShowTableCellComponent content="Hello World!" />);
    const result = <td style={{ fontSize: 25 }}>Hello World!</td>;
    expect(cell.contains(result)).toMatchSnapshot();
  });
});
