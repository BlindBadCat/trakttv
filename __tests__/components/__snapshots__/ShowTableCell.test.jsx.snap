// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShowTableCellComponent /> Should render content 1`] = `true`;
